app/src/utils/data/contributions.ts

```diff
@@ -308,7 +309,7 @@ export function grantDonationListener(
           inRounds: grantRounds,
           grantAddress: args.grantPayees[grantId],
           address: tx.from,
-          donationToken: args.donationToken,
+          donationToken: { ...args.donationToken },
```

Comment:
Attempting to save vue wrapped objects/primitives into `indexedDB` throws some nasty errors!